templates/notification/unsubscribeNotificationsForm.tpl

```diff
@@ -0,0 +1,49 @@
+{**
+ * templates/notification/unsubscribeNotificationsForm.tpl
+ *
+ * Copyright (c) 2013-2019 Simon Fraser University
+ * Copyright (c) 2003-2019 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
```

Comment:
Another GPLv2 -- watch out for these! (Also copyright dates.)

Reply:
Fixed